frontend/src/pages/[user]/index.jsx

```diff
@@ -50,15 +51,22 @@ UserPage.propTypes = {
 }
 
 const User = ({ username }) => {
-  const { data: userProjects } = useSWR(username, fetchUserProjects)
+  const { data, setSize } = useSWRInfinite(
+    getKey('*', `ownerName = ${username}`),
+    gridFetcher,
+  )
+  const intersectionObserverRef = useUpdateBeforeReachingLimit(setSize)
+
+  const userProjects = data?.flat()
```

Comment:
Does it need `flat()`? I think it would be good to use typescript types to clear up what's happening here.